src/engine/analyzer.js

```diff
@@ -29,9 +29,19 @@ async function analyzeDataPoint(
       positionData.tickCurr <= positionData.tickLeft) &&
     updateAlertStatus(positionId, OUT_OF_BOUNDS_ALERT)
   ) {
-    logger.info("Position Out of Limits", positionData.tickCurr);
+    logger.info(
+      "Position",
+      positionId,
+      " is out of limits",
+      "Left:",
+      positionData.tickLeft,
+      "Right:",
+      positionData.tickRight,
+      "Current:",
+      positionData.tickCurr
+    );
     await notify(
-      `Position Out of Limits: ${positionData.tickCurr}`,
+      `Position ${positionId} is out of limits: Left: ${positionData.tickLeft}, Right: ${positionData.tickRight}, Curr: ${positionData.tickCurr}`,
```

Comment:
need to changed to position object ({id: x, chain: y})

Reply:
as for this is the convention in our code

Reply:
this is actually only the id of the positioon (no chain)